### PR TITLE
refactor(packets): Introduce _pad and _padToLength to avoid repetition

### PR DESCRIPTION
**Pull request by ProbablePrime**
I'd love to try and make page 1 cleaner but when i started using these new methods it got large quick

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const NUM_FIRST_PAGE_PIXELS = 2583;
 const NUM_SECOND_PAGE_PIXELS = 2601;
 const ICON_SIZE = 72;
 const NUM_TOTAL_PIXELS = NUM_FIRST_PAGE_PIXELS + NUM_SECOND_PAGE_PIXELS;
+
```
Comment (ProbablePrime) [inline index.js]: added a space here to separate constants from init code

File: index.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const NUM_FIRST_PAGE_PIXELS = 2583;
...
 const keyState = new Array(NUM_KEYS).fill(false);
```
Comment (ProbablePrime) [inline index.js]: D: we should move this into the class for easier expansion when we allow multiple decks.

Comment (ProbablePrime) [inline index.js]: Done in #16

**Comment by Lange** [comment]
This also LGTM, good to merge :100:

**Pull request merged**